check-prod-health-host.sh

```diff
@@ -0,0 +1,6 @@
+HOST=${1}.service.core-compute-prod.internal
```

Comment:
I don't want to sound pedantic, but wouldn't it be better to use an array of ip addresses and a loop?

Reply:
YAGNI, this is a very simple script built in vim,
```
yy
pp
cw
```